ext_localconf.php

```diff
@@ -13,7 +12,25 @@
 $GLOBALS['TYPO3_CONF_VARS']['BE']['flexFormXMLincludeDiffBase'] = true;
 
 $renderFceHeader = '';
-if ($_EXTCONF['enable.']['renderFCEHeader']) {
+
+if (version_compare(TYPO3_version, '9.5.0', '>=')) {
```

Comment:
1.) This is no CodingStyle change ;-)
2.) Please use Tvp\TemplaVoilaPlus\Service\ConfigurationService::getExtensionConfig()
3.) The constructor needs this updates you wrote here

Reply:
Extracted as requested